controllers/meshsync_controller.go

```diff
@@ -93,13 +99,46 @@ func (r *MeshSyncReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 func (r *MeshSyncReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&mesheryv1alpha1.MeshSync{}).
+		Watches(&source.Kind{Type: &corev1.Service{}}, &handler.EnqueueRequestForObject{}).
```

Comment:
@Revolyssup instead of reconciling MeshSync we should reconcile Broker.

Reply:
Let’s go back to the drawing board here. First, an example of is similar implementation from another operator. Second, support for Meshery Broker service networking configuration. Zeroth, fixes for our broken Meshery adapters.